cmake/LibFindMacros.cmake

```diff
@@ -52,6 +52,10 @@ function (libfind_pkg_detect PREFIX)
   if (libraryargs)
     find_library(${PREFIX}_LIBRARY NAMES ${libraryargs} HINTS ${${PREFIX}_PKGCONF_LIBRARY_DIRS})
   endif()
+  # Read pkg-config version
+  if (${PREFIX}_PKGCONF_VERSION)
+    set(${PREFIX}_VERSION ${${PREFIX}_PKGCONF_VERSION} PARENT_SCOPE)
+  endif()
```

Comment:
should this be sent to https://github.com/Tronic/cmake-modules so that we don't miss it next time this file is updated?

Reply:
It is coming from there.

Reply:
Was updated/merged only recently (after I've pulled the last update).